### PR TITLE
added verlet integration

### PR DESCRIPTION
**Pull request by Ved-Mahajan**

### CODE REVIEW

**Comment by Ved-Mahajan** [comment]
This is really strange lol

It was working earlier

File: src/math/sieve_of_eratosthenes.jl [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (Panquesito7) [inline src/math/sieve_of_eratosthenes.jl:1]:
```suggestion
```

File: src/math/sieve_of_eratosthenes.jl [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+"""
+Sieve of Eratosthenes is an algorithm for finding all the primes upto a limit `n`.
+Reference:
+-https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes
```
Comment (Panquesito7) [inline src/math/sieve_of_eratosthenes.jl:5]:
```suggestion
-https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes

Contributed By:- [Ved Mahajan](https://github.com/Ved-Mahajan)
```

File: src/math/verlet.jl [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- `x0` : the initial condition. This is a Vector with the first element as initial value for position(x_0) and the the second  initial condition for velocity (v_0) 
+- `tspan`: is the time span for integration. It is a tuple (initial time, final time)
+This functionr returns a tuple (x,t):
+- `x` is the solution
+- `t` is the array containing the time points
+
+Reference:
+- https://www.algorithm-archive.org/contents/verlet_integration/verlet_integration.html
+
+Contributed by: [Ved Mahajan](https://github.com/Ved-Mahajan)
```
Comment (Panquesito7) [inline src/math/verlet.jl:14]:
```suggestion
Contributed By:- [Ved Mahajan](https://github.com/Ved-Mahajan)
```

File: src/math/sieve_of_eratosthenes.jl [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+"""
+function eratosthenes(n)
```
Comment (Panquesito7) [inline src/math/sieve_of_eratosthenes.jl:7]: Please add documentation for this function.

File: test/math.jl [hunk 5]
```diff
@@ -228,4 +228,13 @@
...
     end
 
+	@testset "Math: Verlet scheme" begin
+		@test verlet_integration(x->-x,[0.0,0.1],(1,10))[end][end] == 9.999	
+	end
+	
+	@testset "Math: Sieve of Eratosthenes" begin
+        @test eratosthenes(20) ==   [2,3,5,7,11,13,17,19]
+        @test eratosthenes(2) == [2]
+        @test eratosthenes(1) == Int64[] # No prime number less that 1, so it should return empty array.
```
Comment (Panquesito7) [inline test/math.jl:238]:
```suggestion
        @test eratosthenes(1) == Int64[] # No prime number less than 1, so it should return an empty array.
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
👍 Good work.

File: src/math/sieve_of_eratosthenes.jl [hunk 6]
```diff
@@ -0,0 +1,18 @@
+"""
+Sieve of Eratosthenes is an algorithm for finding all the primes upto a limit `n`.
+Reference:
+-https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes
+
+Contributed By:- [Ved Mahajan](https://github.com/Ved-Mahajan)
+"""
+function eratosthenes(n)
```
Comment (Panquesito7) [inline src/math/sieve_of_eratosthenes.jl:8]: This function is missing documentation. Please add documentation to the function.

**Comment by Ved-Mahajan** [comment]
It seems that there is some mess on my side.

I am closing this PR and will make a new PR with all the stuff in place

**Pull request closed without merging**